src/main/java/gov/cabinetofice/gapuserservice/service/SpotlightService.java

```diff
@@ -125,16 +129,62 @@ public void exchangeAuthorizationToken(String authorizationCode, String state) t
             log.debug("new Access token: {}", accessToken);
             log.debug("new Refresh token: {}", refreshToken);
 
-            this.updateSecret("access_token", accessToken);
-            this.updateSecret("refresh_token", refreshToken);
+            this.updateSecret(ACCESS_TOKEN_NAME, accessToken);
+            this.updateSecret(REFRESH_TOKEN_NAME, refreshToken);
 
         } catch (IOException e) {
```

Comment:
Can do this as both exceptions are handled in the same way:

`catch(IOException | JSONException e)`

Reply:
Fixed